lib/stack_master/commands/nag.rb

```diff
@@ -0,0 +1,28 @@
+module StackMaster
+  module Commands
+    class Nag
+      include Command
+      include Commander::UI
+
+      def perform
+        Tempfile.open(['stack', "___#{stack_definition.stack_name}.#{proposed_stack.template_format}"]) do |f|
+          f.write(proposed_stack.template_body)
+          f.flush
+          system('cfn_nag', f.path)
```

Comment:
How do we pass the exit code back out? When running in CI we'd want it to exit non-zero if the checks fail.

Reply:
Good catch @patrobinson 🤦 

Fixed in 46e5db7, including tests (which are a little weird, happy to hear other mechanisms for testing/mocking exit status).

I will note that I based my implementation on the `lint` command, which has no such behavior.  Probably worth a PR to fix that as well.

Reply:
This is unfortunately because the lint command exits with `0` only. :(

Reply:
Right, that makes sense.  I wonder if it would be reasonable to perform a string match on the output (assuming there's some sentinel value we could use to identify failure/success) and use that to exit properly? 

In any case, that's work for a future PR.